app/utils.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import annotations
+
+import json
+import os
+import shutil
+from datetime import datetime
+from pathlib import Path
+from typing import TYPE_CHECKING, Any
+
+import yaml
+from checkov.common.bridgecrew.wrapper import reduce_scan_reports
+from flask import current_app as webhook
+from flask import jsonify
+
+from app.consts import MANIFEST_ROOT_PATH, WHORF_CONFIG_PATH
+from app.models import WhorfConfig
+
+if TYPE_CHECKING:
+    from checkov.common.output.report import Report
+    from flask import Response
+
+
+def admission_response(*, allowed: bool, uid: str, message: str) -> Response:
+    return jsonify(
+        {
+            "apiVersion": "admission.k8s.io/v1",
+            "kind": "AdmissionReview",
+            "response": {
+                "allowed": allowed,
+                "uid": uid,
+                "status": {"code": 403, "message": message},
```

Comment:
@gruebel Is the status supposed to be hard coded to 403? Sounds very negative 🤷

Reply:
this was from @eurogig originally, I'm fine with either making configurable, if needed

Reply:
I'd do something like this:
```suggestion
                "status": {"code": 403 if not allowed else 200, "message": message},
```

Reply:
lets go with that, we can change it, if needed.